spec/attribute.dd

```diff
@@ -283,9 +290,19 @@ struct S
     align (16) short c; // placed at offset 16
 }
 static assert(S.alignof == 16);
+static assert(S.c.offsetof == 16);
 static assert(S.sizeof == 32);
 --------
+)
+        $(P The $(I AlignAttribute) is reset to the default when
```

Comment:
I think this is better worded as "xyz is not affected by" rather than "is reset to default when xyz is entered"